lib/variations.js

```diff
@@ -17,7 +20,7 @@ const variations = {
 };
 
 function isEligible() {
-  return true;
+  return !Preferences.isSet('services.sync.username'); // users with sync are ineligible
```

Comment:
I placed a utility function in `Advisor.js` for internal use, but it can't be called by `isEligible` because the Advise object is null at the time that it executes.